### PR TITLE
adding mockinterview platform

### PR DESCRIPTION
**Pull request by prakashn27**
fixing duplicate issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,12 +152,12 @@
 ### Mock interviews
 
 * [interviewing.io](https://interviewing.io/)
+* [mockinterview.app](https://mockinterview.app)
 * [Pramp](https://www.pramp.com/) *"Practice coding interviews for free"*
 * [Refdash](https://refdash.com/)
 * [Gainlo](http://www.gainlo.co/)
 * [Candidacy.io](https://www.candidacy.io/)
 * [Skilled](https://www.skilledinc.com/)
-* [Gainlo](http://www.gainlo.co/#!/)
```
Comment (Olshansk) [inline README.md:160]: Why did you remove this?

**Review by Olshansk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (prakashn27) [reply README.md:160]: there is already a Gainlo in line 158.
